src/Veldrid.ImageSharp/ImageSharpCubeMapTexture.cs

```diff
@@ -0,0 +1,105 @@
+﻿using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Advanced;
+using SixLabors.ImageSharp.PixelFormats;
+using System;
+using System.Runtime.InteropServices;
+
+namespace Veldrid.ImageSharp
+{
+    public class ImageSharpCubeMapTexture
+    {
+    
+        /// <summary>
+        /// An array of images, each face of a cubemap.
+        /// </summary>
+        public Image<Rgba32>[] CubeMapTextures { get; }
+
+        /// <summary>
+        /// The width of a cubemap texture.
+        /// </summary>
+        public uint Width => (uint)CubeMapTextures[0].Width;
+
+        /// <summary>
+        /// The height of a cubemap texture.
+        /// </summary>
+        public uint Height => (uint)CubeMapTextures[0].Height;
+
+        /// <summary>
+        /// The pixel format cubemap textures.
+        /// </summary>
+        public PixelFormat Format => PixelFormat.R8_G8_B8_A8_UNorm;
+
+        /// <summary>
+        /// The size of each pixel, in bytes.
+        /// </summary>
+        public uint PixelSizeInBytes => sizeof(byte) * 4;
+
+        /// <summary>
+        /// Provides standardized access to the cubemap texture array
+        /// </summary>
+        private const int FRONT = 0;

```

Comment:
Additionally, since these are the order they appear in the public array, we should make these values correspond to the texture array layers that they end up in. The order is:

+X
-X
+Y
-Y
+Z
-Z

Perhaps those are better names for these constants anyways, since front/back implies a handedness convention.